roles/lib/files/FWO.Report/ReportCompliance.cs

```diff
@@ -552,6 +606,31 @@ private void TryAppendCsvHeader(StringBuilder sb, List<string> propertyNames)
             }
         }
 
+        private List<NetworkObject> GetAllNetworkObjectsFromRule(Rule rule)
```

Comment:
this sounds like a much more general method that should probably located not only in compliance (and very likely already exists somewhere in the code) - but where?